### PR TITLE
Feature/credit notes 

### PR DESCRIPTION
**Pull request by SimonSchlossarek**

### CODE REVIEW

**Comment by SimonSchlossarek** [comment]
adding new credit note download

File: src/Api/Invoice.php [hunk 1]
```diff
@@ -110,44 +110,9 @@ public function allAfter(int $timestamp)
...
-                'unity' => [
-                    'id' => 1,
-                    'objectName' => 'Unity',
-                ]
-            ]
-        ];
-        $allParameters = array_replace_recursive($requiredParameters, $parameters);
-        return $this->_post(Routes::CREATE_INVOICE, $allParameters);
+        return $this->_post(Routes::CREATE_INVOICE, $parameters);
     }
```
Comment (exlo89) [inline src/Api/Invoice.php:116]: can you explain why you delete the requiredParameters.

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "exlo89/laravel-sevdesk-api",
+    "name": "sq/laravel-sevdesk-api",
     "description": "A helpful Sevdesk API client for Laravel.",
```
Comment (exlo89) [inline composer.json:3]: please change it back

File: src/Api/Utils/ApiClient.php [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <?php
 /*
  * ApiClient.php
- * @author Martin Appelmann <hello@martin-appelmann.de>
- * @copyright 2021 Martin Appelmann
+ * @author Simon Schlossarek
+ * @copyright 2023 Simon Schlossarek
  */
```
Comment (exlo89) [inline src/Api/Utils/ApiClient.php:7]: also here. please change it back. If you code a significant part of this project you will become a contributor.

**Review by exlo89 — COMMENTED** [review]
please check the comments

**Pull request closed without merging**